secant/UI Components/CheckCircle/CheckCircleStore.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  CheckCircleStore.swift
+//  secant-testnet
+//
+//  Created by Michal Fousek on 26.09.2022.
+//
+
+import Foundation
+import ComposableArchitecture
+import SwiftUI
+
+typealias CheckCircleReducer = Reducer<CheckCircleState, CheckCircleAction, CheckCircleEnvironment>
+typealias CheckCircleStore = Store<CheckCircleState, CheckCircleAction>
+typealias CheckCircleViewStore = ViewStore<CheckCircleState, CheckCircleAction>
+
+// MARK: - State
+
+struct CheckCircleState: Equatable {
+    var isChecked: Bool
+}
+
+// MARK: - Action
+
+enum CheckCircleAction: Equatable {
+    case updateIsChecked
+}
+
+// MARK: - Environment
+
+struct CheckCircleEnvironment { }
+
+extension CheckCircleEnvironment {
+    static let live = CheckCircleEnvironment()
+    static let mock = CheckCircleEnvironment()
+}
+
```

Comment:
Similarly, such primitive component as toggle will never need any environment (dependency). Use Void instead. That way we can cleanup the code which also reduces the instances of `CheckCircleEnvironment()`

Combined with the state comment, use:
Use `typealias CheckCircleReducer = Reducer<bool, CheckCircleAction, Void>`